src/renderer/components/AircraftSection/Properties/index.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
```

Comment:
2:10  error  'Link' is defined but never used       @typescript-eslint/no-unused-vars